docs/usage/api.rst

```diff
@@ -1,87 +1,59 @@
 API overview
 ------------
 
-RestrictedPython has tree major scopes:
-
-1. ``compile_restricted`` methods:
+``compile_restricted`` methods
+++++++++++++++++++++++++++++++
 
 .. py:method:: compile_restricted(source, filename, mode, flags, dont_inherit, policy)
     :module: RestrictedPython
 
     Compiles source code into interpretable byte code.
 
-    :param source: (required). The source code that should be compiled
-    :param filename: (optional).
-    :param mode: (optional).
-    :param flags: (optional).
-    :param dont_inherit: (optional).
-    :param policy: (optional).
-    :type source: str or unicode text
+    :param source: (required). the source code that should be compiled
+    :param filename: (optional). defaults to ``'<unknown>'``
+    :param mode: (optional). defaults to ``'exec'``
+    :param flags: (optional). defaults to ``0``
+    :param dont_inherit: (optional). defaults to ``False``
+    :param policy: (optional). defaults to ``RestrictingNodeTransformer``
+    :type source: str or unicode text or ``ast.AST``
     :type filename: str or unicode text
     :type mode: str or unicode text
     :type flags: int
     :type dont_inherit: int
     :type policy: RestrictingNodeTransformer class
-    :return: Byte Code
+    :return: Python byte code
```

Comment:
Is there a Python class for `Python byte code` or is it not possible to specify the type of the return value?